src/Agent.Worker/Variables.cs

```diff
@@ -169,6 +169,20 @@ public Variables(IHostContext hostContext, IDictionary<string, VariableValue> co
 
         public string System_TFCollectionUrl => Get(WellKnownDistributedTaskVariables.TFCollectionUrl);
 
+        public static readonly HashSet<string> PiiVariables = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        {
+            "Build.AuthorizeAs",
```

Comment:
What about all `releases.` variables?

Reply:
This is a very good point. So far, I've only looked at the build documentation, and I forgot about release. I'll do so as my next task.